src/ResourceManager/Resources/AzureRM.Resources.Netcore.psd1

```diff
@@ -119,7 +120,12 @@ CmdletsToExport = 'Get-AzureRmProviderOperation', 'Remove-AzureRmRoleAssignment'
     'Set-AzureRmManagedApplicationDefinition', 
     'Remove-AzureRmManagedApplicationDefinition', 
     'Get-AzureRmManagedApplication', 'New-AzureRmManagedApplication', 
-    'Set-AzureRmManagedApplication', 'Remove-AzureRmManagedApplication'
+    'Set-AzureRmManagedApplication', 'Remove-AzureRmManagedApplication',
+    'Get-AzureRmManagedApplication', 'New-AzureRmManagedApplication', 
```

Comment:
It looks like these four cmdlets got repeated for some reason.  Please fix.

Reply:
Apologies for that will fix it.